data/binding/gen.go

```diff
@@ -675,6 +677,7 @@ import "fyne.io/fyne/v2"
 		bindValues{Name: "Int", Type: "int", Default: "0", Format: "%d", SupportsPreferences: true},
 		bindValues{Name: "Rune", Type: "rune", Default: "rune(0)"},
 		bindValues{Name: "String", Type: "string", Default: "\"\"", SupportsPreferences: true},
+		bindValues{Name: "Untyped", Type: "interface{}", Default: "struct{}{}", Since: "2.1"},
```

Comment:
Any particular reason to have the default value set to "struct{}{}" instead of "nil"?

Reply:
No, good catch. It mattered in earlier versions of this code but should have been fixed for PR.
Updated now